lms/templates/cme_register.html

```diff
@@ -12,6 +12,15 @@
 <%! from datetime import date %>
 <%! import calendar %>
 
+<%
+  if self.theme_enabled() and not self.is_microsite():
+    honor_link = marketing_link('TOS') + "#honor"
+    register_sidebar_file = 'theme-register_sidebar.html'
+  else:
+    honor_link = None
```

Comment:
Instead of defining these in an `else` block, we can have cleaner code by assigning a default unconditionally.

```python
honor_link = None
register_sidebar_file = None
if self.is_non_microsite_theme():
    honor_link = marketing_link('TOS') + '#honor'
    register_sidebar_file = 'theme-register_sidebar.html'
```